### PR TITLE
Bdog 1707

### PR DESCRIPTION
**Pull request by colin-lamed**

### CODE REVIEW

File: init_service/templates/service/conf/application.conf [hunk 1]
```diff
@@ -50,39 +25,7 @@ microservice {
...
-
-  uk.gov.hmrc.govukfrontend.controllers.Assets = {
-    needsAuditing = false
-  }
-
-  uk.gov.hmrc.hmrcfrontend.controllers.Assets = {
-      needsAuditing = false
-  }
-}
+metrics.enabled = true
```
Comment (colin-lamed) [inline init_service/templates/service/conf/application.conf:28]: will remove this later once it's added to app-config-common

File: init_service/templates/service/conf/application.conf [hunk 2]
```diff
@@ -174,28 +100,10 @@ controllers {
...
 # Metrics plugin settings - graphite reporting is configured on a per env basis
-metrics {
-  name = ${appName}
-  rateUnit = SECONDS
-  durationUnit = SECONDS
-  showSamples = true
-  jvm = true
-  enabled = true
-}
+metrics.enabled = true
```
Comment (colin-lamed) [inline init_service/templates/service/conf/application.conf:103]: will remove this later when once added to app-config-common

File: init_service/templates/service/conf/application.conf [hunk 3]
```diff
@@ -206,9 +114,6 @@ microservice {
   metrics {
     graphite {
       host = graphite
```
Comment (colin-lamed) [inline init_service/templates/service/conf/application.conf:116]: will remove this later once added to app-config-common

**Review by RikSherman — APPROVED** [review]

**Comment by chotai** [comment]
Would you mind updating the Play plugin to `2.8.19` to match the version of Play in bootstrap-play please (the current mismatch results in runtime errors)

**Review by chotai — APPROVED** [review]

**Pull request merged**